core/controllers/dashboard.py

```diff
@@ -166,6 +167,8 @@ def _get_intro_card_color(category):
 
             feedback_thread_analytics = feedback_services.get_thread_analytics(
                 exp_summary.id)
+            exp_stats = stats_services.get_exploration_stats(
+                exp_summary.id, 'all')
             # TODO(sll): Reuse _get_displayable_exp_summary_dicts() in
```

Comment:
@seanlip PTAL: https://github.com/oppia/oppia/pull/2096/commits/e313f226e565e67e47eb56d7a614c20ee6996884
(Also, if this approach is fine then I think I can implement this for the collection summary dicts as well, as mentioned in the TODO below this one)